--- gtkglext-1.2.0/gdk/gdkglshapes.c	2018-07-15 21:43:11.220428889 +0200
+++ gtkglext-1.2.0/gdk/gdkglshapes.c	2006-02-05 04:25:00.000000000 +0100
@@ -544,7 +544,7 @@
   {-Z, -X, 0}
 };
 
+static int myindex[20][3] =
-static int index[20][3] =
 {
   {0, 4, 1},
   {0, 9, 4},
@@ -574,7 +574,7 @@
   int i;
 
   for (i = 19; i >= 0; i--) {
+    drawtriangle(i, idata, myindex, shadeType);
-    drawtriangle(i, idata, index, shadeType);
   }
 }
 
